core/components/minishop2/model/minishop2/mscarthandler.class.php

```diff
@@ -132,8 +132,10 @@ function __construct(miniShop2 & $ms2, array $config = array())
      */
     public function initialize($ctx = 'web')
     {
+        if ($this->modx->getOption('ms2_cart_context', null, '', true) == 1){
```

Comment:
if ($this->modx->getOption('ms2_cart_context', null, '', true)){
            $ctx = 'web';
        }